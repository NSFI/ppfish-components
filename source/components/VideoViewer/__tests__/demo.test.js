import demoTest from '../../../tests/demoTest';

demoTest('VideoViewer');
