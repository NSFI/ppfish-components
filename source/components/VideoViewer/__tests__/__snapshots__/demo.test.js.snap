// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders videoViewer demo correctly 1`] = `
<div>
  <div
    class="fishd-video-js-wrap"
  >
    <div
      data-vjs-player="true"
    >
      <video
        class="video-js"
      />
    </div>
  </div>
  <div
    class="fishd-video-js-wrap"
  >
    <div
      data-vjs-player="true"
    >
      <video
        class="video-js"
      />
    </div>
  </div>
  <div
    class="source"
  >
    <div
      class="block"
    >
      <div
        class="fishd-video-viewer-wrap"
        style="width:240px;height:135px"
      >
        <div
          class="fishd-video-viewer-thumb"
        >
          <div
            class="fishd-video-viewer-thumb-status fishd-video-viewer-thumb-big-play-button"
          >
            <i
              class="fishdicon fishdicon-play"
            />
          </div>
          <div
            class="fishd-video-js-wrap"
          >
            <div
              data-vjs-player="true"
            >
              <video
                class="video-js"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="block"
    >
      <div
        class="fishd-video-viewer-wrap"
        style="width:240px;height:135px"
      >
        <div
          class="fishd-video-viewer-thumb"
        >
          <div
            class="fishd-video-viewer-thumb-status fishd-video-viewer-thumb-big-play-button"
          >
            <i
              class="fishdicon fishdicon-play"
            />
          </div>
          <div
            class="fishd-video-js-wrap"
          >
            <div
              data-vjs-player="true"
            >
              <video
                class="video-js"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="source"
  >
    <div
      class="block"
    >
      <div
        class="fishd-video-viewer-wrap"
        style="width:240px;height:135px"
      >
        <div
          class="fishd-video-viewer-thumb fishd-video-viewer-thumb-disabled"
        >
          <div
            class="fishd-video-viewer-thumb-status fishd-video-viewer-thumb-failed-message"
          >
            <span>
              已过期
            </span>
          </div>
          <div
            class="fishd-video-js-wrap"
          >
            <div
              data-vjs-player="true"
            >
              <video
                class="video-js"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="block"
    >
      <div
        class="fishd-video-viewer-wrap"
        style="width:240px;height:135px"
      >
        <div
          class="fishd-video-viewer-thumb fishd-video-viewer-thumb-disabled"
        >
          <div
            class="fishd-video-viewer-thumb-status fishd-video-viewer-thumb-failed-message"
          >
            <span>
              状态描述
            </span>
          </div>
          <div
            class="fishd-video-js-wrap"
          >
            <div
              data-vjs-player="true"
            >
              <video
                class="video-js"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="source"
  >
    <div
      class="block"
    >
      <button
        class="fishd-btn fishd-btn-primary"
        type="button"
      >
        <span>
          点击播放视频
        </span>
      </button>
    </div>
  </div>
</div>
`;
